src/main/java/com/google/devtools/build/lib/remote/disk/DiskCacheClient.java

```diff
@@ -43,20 +45,27 @@ public class DiskCacheClient implements RemoteCacheClient {
 
   private final Path root;
   private final boolean verifyDownloads;
+  private final boolean checkActionResult;
   private final DigestUtil digestUtil;
 
-  public DiskCacheClient(Path root, boolean verifyDownloads, DigestUtil digestUtil) {
+  public DiskCacheClient(Path root, boolean verifyDownloads, boolean checkActionResult,
```

Comment:
Add a Javadoc with an explanation of what `checkActionResults` means, so that readers don't need to look for this PR to understand it. (Optional: also for `verifyDownloads`.)

Reply:
Done.